ktor-client/ktor-client-cio/common/src/io/ktor/client/engine/cio/CIOEngine.kt

```diff
@@ -137,6 +137,7 @@ internal class CIOEngine(
 @Suppress("KDocMissingDocumentation")
 @Deprecated(
     "Use ClientEngineClosedException instead",
+    level = DeprecationLevel.ERROR,
```

Comment:
can be an error in 1.6.0 and removed in 2.0.0

Reply:
But would it be breaking change?